cli/flags.rs

```diff
@@ -1068,6 +1070,7 @@ Ignore formatting a file by adding an ignore comment at the top of the file:
   // deno-fmt-ignore-file",
     )
     .arg(config_arg())
+    .arg(no_config_arg())
```

Comment:
It's slightly error prone to have to provide both of these each time. Is there a way we could combine them into one function that adds both args at the same time and force the use of that? I think we do that elsewhere IIRC.